.github/workflows/rerun.yml

```diff
@@ -0,0 +1,25 @@
+# Cross version tests sometimes fail due to transient errors. This workflow reruns failed tests.
+name: rerun-cross-version-tests
+
+on:
+  schedule:
+    # Run this workflow daily at 17:00 UTC (4 hours after cross-version-tests.yml workflow)
+    - cron: "0 17 * * *"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
```

Comment:
This will force-abort a manual execution of the cross version tests, right? Do we want to do that?

Reply:
It depends on when we run workflow.

Reply:
We have a `conclusion === "success"` guard, but we should not hit that.